functions/access.js

```diff
@@ -18,19 +18,21 @@ const checkIsValidToken = async (venueId, uid, token) => {
         transaction.get(accessRef),
       ]);
 
-      if (!venue.exists || !granted.exists || !granted.tokens) {
+      console.log("asdasd granted", granted);
```

Comment:
These `console.logs` should be removed before this PR is ready to be merged. (i'll only point this out once, but they were in many of the files, so worth reviewing them all)